### PR TITLE
Distribute sql files within war file.

### PR DESCRIPTION
**Pull request by henning-gerhardt**
Distribute sql files (schema.sql and default.sql) within generated war file for easier installation process.

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -196,6 +196,8 @@
         <mkdir dir="${test.report.dir}"/>
     	<mkdir dir="${build.dir.plugins.import}/PicaMassImport"/>
         <mkdir dir="${build.dir.plugins.opac}/PicaPlugin"/>
+        <mkdir dir="${build.dir}/scripts"/>
```
Comment (stweil) [inline build.xml]: This is not needed in my ANT build.

**Comment by stweil** [comment]
Is it possible that more setup related files would be useful in the future, for example scripts which help during a migration from an old release to a newer one? Then I'd keep the name `setup`.

**Comment by henning-gerhardt** [comment]
I don't think that migration files should be provided in the war file. I accepted your PR and created this PR for easier new installations. 

Migration of any kind of data and structure should not applied automatically.

**Comment by stweil** [comment]
Migration scripts were only an example. We could also add scripts which create the SQL database or which check whether all required software packages are installed and install those that are missing, so there are lots of use cases for scripts needed for new installations.

**Pull request merged**